epub33/rs/index.html

```diff
@@ -1789,10 +1805,9 @@ <h2>Security and Privacy</h2>
 			<section id="sec-scripted-content-security">
 				<h4>Scripting Considerations</h4>
 
-				<p><a>EPUB Creators</a> and Reading System developers must be aware of the security issues that arise
-					when Reading Systems execute scripted content. As the underlying scripting model employed by Reading
-					Systems and browsers is the same, developers must take into consideration the same kinds of issues
-					encountered in Web contexts.</p>
+				<p><a>EPUB Creators</a> and Reading System developers who also support scripting  must be aware of the 
```

Comment:
We don't really need "epub creators" in this context anymore. It's the RS spec.

Reply:
Correct. Removed it in https://github.com/w3c/epub-specs/pull/1671/commits/86beb17f4004082c39aeaf05537832ad23fad694